src/static/helperTemplates/slas.ts.hbs

```diff
@@ -0,0 +1,423 @@
+/*
```

Comment:
This looks good but I feel this should be a helper instead of a template. Helpee can accept the instance of the shopperLogin class